seabird/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "seabird" %}
```

Comment:
this recipe needs to be in `recipes/seabird/meta.yaml`

Reply:
Ouch! Where is my mind? I'm sorry about that.